TwicketSegmentedControl/TwicketSegmentedControl.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 
 public protocol TwicketSegmentedControlDelegate: class {
-    func didSelect(_ segmentIndex: Int)
+    func twicketSegmentedControl(_ segmentedControl: TwicketSegmentedControl, didSelect segmentIndex: Int)
```

Comment:
What do you think about `didSelect(_ segmentIndex: Int, on segmentedControl TwicketSegmentedControl)` ?